scripts/package-release-assets.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+mkdir release
+cd artifacts || exit
```

Comment:
I think you can remove this check.
```suggestion
cd artifacts
```

And add `set -e` to the start of the script, so that it exits on failure of any command (including `cd`).